MangoAPI.Client/src/app/components/chats/chats.component.html

```diff
@@ -202,7 +201,19 @@ <h5>Chats</h5>
                   <img alt="User Avatar" class="user-avatar" src="{{message.messageAuthorPictureUrl}}"/>
                   <div class="message-wrapper">
                     <div class="message-content">
-                      <div class="display-name">{{message.userDisplayName}}</div>
+                      <div class="display-name"
+                           [ngClass]="{
+                                        'color-white': message.userDisplayNameColour === displayNameColours.White, 
```

Comment:
Please move it to separate method or anything else, layout is not a place for logic